src/DynamoCoreWpf/Views/PackageManager/Components/PackageManagerWizard/PackageManagerWizard.xaml.cs

```diff
@@ -330,7 +340,11 @@ private async void SendMarkdownFilesDirectory(string path)
 
         private async void SendErrorString(string error)
         {
-            if (string.IsNullOrEmpty(error)) return;
+            // We only want to surface actual publish Errors to the front end,
+            // and 'Ready to publish' is triggered multiple times during the publishing process
+            // preventing us to report an actual Error.
+            if (error.Equals(Wpf.Properties.Resources.PackageManagerReadyToPublish))
```

Comment:
why is PackageManagerReadyToPublish thrown as error?

Reply:
Renamed to `PackageManagerNoValidationErrors` for clarity.